src/commandlineparser.cpp

```diff
@@ -49,6 +50,10 @@ int CommandLineParser::parse(int argc, char* argv[]) {
     tokens.append(QString(argv[i]));
   }
 
+  QCoreApplication::setApplicationName("Mozilla VPN");
```

Comment:
The impetus for this change was to remove the hardcoded organization name in `SettingsHandler` so that it could be switched from production to testing when we are running in test. This particular addition here in `CommandLineParser::parse()` sets the default value, when can get overwritten later (eg: if `--testing` is set).

It wasn't sufficient to call `setOrganizationName()` in the old location, as we need to do it before the attempting to load the settings file. So it wound up here (I suppose it could have been moved even earlier, like into `main()` or something too).

The rest of the values were added purely for code reuse purposes.